packages/oscal-react-library/src/components/images/icons/fileUpload.svg:Zone.Identifier

```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]
+ZoneId=3
+HostUrl=https://www.figma.com/

```

Comment:
Same comment here on the `svg`s as #927: we want these to be `svg`s with path code instead of figma links.